src/k8s/pkg/k8sd/api/cluster_bootstrap.go

```diff
@@ -40,7 +40,7 @@ func postClusterBootstrap(m *microcluster.MicroCluster, s *state.State, r *http.
 	}
 
 	// Set timeout
-	timeout := utils.TimeoutFromCtx(s.Context, 30*time.Second)
+	timeout := utils.TimeoutFromCtx(s.Context, 60*time.Second)
```

Comment:
That is not the correct context. Slipped through in the review. cc @louiseschmidtgen 
It should be `r.Context()`. This context includes a timeout, so the default value is not used. Could you please also check that the other occurrences of `TimeoutFromCtx` use the correct context? Thanks!

Reply:
@bschimke95 I've changed the context the `r.Context()` but the default value still seems to be used as I got test failures, but  bumping the default timeout to 45 seconds gets me a passing CI. I don't have more background on this, would appreciate if you could take a look into it as well.